.github/Style Guide.md

```diff
@@ -0,0 +1,227 @@
+
+## Branch titles <a name="branch"></a>
+Branches that are related to the content of the Sector File itself (99% of the issues) should simply be titled `issue-{issue number}`.
+<br>✅ `issue-4427`
+<br>❌ `delete-luton-runways`
+
+Where the issue is not a standard one, or no issue exists at all (such as for documentation relating to the SF or updates to the GH workflows) then a brief summary of the changes will suffice as a branch title. These will generally only by used by maintainers. 
+This should be in all lowercase, with words separated by hyphens.
+<br>✅`compiler-1.1.2`
+<br>✅`readme-formatting`
+<br>❌`LOTSOFCHANGELOGCHANGESthatmakepeoplehappy`
+
+## General rules <a name="general"></a>
+These rules apply to all the stated areas.
+
+Writing should be written in [sentence-case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case):
+<br>✅ Issue: `GAGSU incorrect spelling`
+<br>❌ Issue: `GAGSU Incorrect Spelling`
+<br>✅ PR: `Fixes #3673 - Correct S24 ownership`
+<br>❌ PR: `Fixes #3673 - Correct S24 Ownership`
+
+Writing should also should omit (where it makes sense to do so):
+
+- Generic words such as 'issue' or 'problem' (and replaced with a more helpful description)
+- Articles such as 'the' and 'an'
+- Auxiliary/superfluous verbs ('are', 'is', 'has')
+
+<br>✅ Issue: `Manchester (EGCC) A1 hold wrong location`
+<br>❌ Issue: `Manchester (EGCC) has an issue with the A1 hold`
+<br>✅ PR: `Fixes #2134 - Correct Manchester (EGCC) A1 hold location`
+<br>❌ PR: `Fixes #2134 - Correct the A1 hold location at Manchester (EGCC)`
+
+
+## Issue titles <a name="issues"></a>
+Where it makes sense to do so, issue titles should omit the main verb:
+<br>✅ `New Fairoaks (EGTF) SMR`
+<br>❌ `Add new Fairoaks (EGTF) SMR`
+
+
+They should be a brief description of either the issue that requires solving, or what the changes would (aim to) do. It should be sufficiently precise without being too verbose (the issue main body should be used to elaborate). A verb does not need to be included unless it reads better.
+<br>✅ `Gatwick (EGKK) runway headings`
+<br>✅ `S23 -> S24 agreements non-functioning`
+<br>❌ `Problems with Gatwick (EGKK) runways`
+<br>❌ `S23 -> S24 altitude agreement at KOKSY doesn't work, tried with various...`
+
+## Pull request titles <a name="prs"></a>
+
+Where pull requests relate to an issue (as 99% of PRs will), then they should begin with `Fixes #{issue number}`:
+<br>✅`Fixes #1234 - Update Gatwick (EGKK) SMR`
+<br>❌`Update Gatwick (EGKK) SMR`
+
+Where they do not, this will of course be omitted.
+
+
+Pull request titles take inspiration from commit message formatting - see [here](https://cbea.ms/git-commit/) for some guidance. The main rules are as follows:

```

Comment:
Going to open a can of worms here - but we could move away from Fixes #x in the names of PRs, because it doesn't actually do anything. That's why we have to repeat the issue number in the PR description. Might make things a little cleaner?!

Reply:
Personally like having issue numbers in PR titles, find it easier to scan for the relevant one.

Reply:
Gonna be honest, I also like the issue names in titles. Maybe it's just a tradition thing but it aesthetically pleases me